pkg/certifier/scorecard/scorecardRunner_test.go

```diff
@@ -24,6 +24,9 @@ import (
 )
 
 func Test_scorecardRunner_GetScore(t *testing.T) {
+	if os.Getenv("GITHUB_EVENT_NAME") != "check_suite" || os.Getenv("GITHUB_EVENT_ACTION") != "completed" {
```

Comment:
@weiherng2000 Once https://github.com/guacsec/guac/pull/906 merges, you could add this to that workflow.